app/src/main/java/org/simple/clinic/forgotpin/confirmpin/ForgotPinConfirmPinEffect.kt

```diff
@@ -6,8 +6,6 @@ object LoadLoggedInUser : ForgotPinConfirmPinEffect()
 
 object LoadCurrentFacility : ForgotPinConfirmPinEffect()
 
-object HideError : ForgotPinConfirmPinEffect()
```

Comment:
Knit pick: The commit message should ideally be - Migrate `HideError` to view effect or Change `HideError` effect to view effect.

Reply:
Not very important to make this change tho

Reply:
Doesn't it already say "Change `HideError` effect as view effect"? 🤔

Reply:
it should be change `to` view effect.

Reply:
`as` basically is a conjunction which showcases a comparison between two things. Like "they can do as they wish" or "Mobius is as good as MVI" but here we are trying to say that we want to change from this to that, therefore it is incorrect to use here.

Reply:
Okay. Will keep this in mind for future commits 👍🏾